### ISSUE TITLE
Slight rewording in MPI version reported

### ISSUE THREAD

**Issue by nilsdeppe** [UIUC-PPL/charm#3584]
Hi folks!

I've been upgrading MPI versions, and I noticed Charm++ prints out:
```
Charm++> Running on MPI version: 3.1
```
This confused me for a couple minutes until I realized it meant the MPI standard version. Maybe rephrase to:
```
Charm++> Running on MPI standard: 3.1
```

**Comment by stwhite91** [comment]
I support the proposed change, and note that we could also print out the MPI library name/version by calling MPI_Get_library_version(). That could help users know whether they built on OpenMPI, Intel MPI, etc.

**Comment by nilsdeppe** [comment]
Having the MPI lib name and version also printed would actually be really fantastic :D

**Comment by rbuch** [comment]
Closed in #3585.
